public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -231,6 +232,10 @@ export function useAllAlertRuleAbilities(rule: CombinedRule): Abilities<AlertRul
       [AlertRuleAction.Silence]: canSilence,
       [AlertRuleAction.ModifyExport]: [isGrafanaManagedAlertRule, exportAllowed],
       [AlertRuleAction.Pause]: [MaybeSupportedUnlessImmutable && isGrafanaManagedAlertRule, isEditable ?? false],
+      [AlertRuleAction.Restore]: toAbility(
+        MaybeSupportedUnlessImmutable && isGrafanaManagedAlertRule,
+        rulesPermissions.create
```

Comment:
Why `create`? Isn't it an update action?

Reply:
good point, i will change it